superset-frontend/src/components/Select/styles.tsx

```diff
@@ -156,6 +156,7 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
           border-color: ${borderColor};
           box-shadow: 0 1px 0 rgba(0, 0, 0, 0.06);
         }
+        flex-wrap: nowrap;
```

Comment:
there are two elements in the container: the wrapper for the inputs and the indicator. We always want these on one row.